LICENSE.md

```diff
@@ -1,31 +1,21 @@
-As a work of the United States government, this project is in the
-public domain within the United States.
+# License
 
-Additionally, we waive copyright and related rights in the work
-worldwide through the CC0 1.0 Universal public domain dedication.
+As a work of the [United States of America's Government](https://www.usa.gov/), this project is in the public domain within the United States of America.
 
-## CC0 1.0 Universal summary
+Additionally, we waive copyright and related rights in the work worldwide through the CC0 1.0 Universal public domain dedication.
+
+## CC0 1.0 Universal Summary
 
 This is a human-readable summary of the [Legal Code (read the full text)](https://creativecommons.org/publicdomain/zero/1.0/legalcode).
 
-### No copyright
+### No Copyright
 
-The person who associated a work with this deed has dedicated the work to
-the public domain by waiving all rights to the work worldwide
-under copyright law, including all related and neighboring rights, to the
-extent allowed by law.
+The person who associated a work with this deed has dedicated the work to the public domain by waiving all of their rights to the work worldwide under copyright law, including all related and neighboring rights, to the extent allowed by law.
```

Comment:
Just noting that the language here is slightly changed from the original, but I'm seeing that it's a gender-neutral version of the [updated CC0 summary](https://creativecommons.org/publicdomain/zero/1.0/), so that's all good. :+1: